zilencer/views.py

```diff
@@ -132,6 +136,31 @@ def register_remote_server(
     except ValidationError as e:
         raise JsonableError(e.message)
 
+    # We don't want to allow disposable domains for contact_email either
+    try:
+        validate_disposable(contact_email)
+    except DisposableEmailError:
+        raise JsonableError(_("Contact email in disposable domains is not allowed"))
+
+    contact_email_domain = Address(addr_spec=contact_email).domain.lower()
+    if contact_email_domain == "example.com":
+        raise JsonableError(_("example.com is not a valid domain for contact email"))
```

Comment:
Let's try to not add new translateable strings here. `_("Invalid address.")` from `zerver/lib/email_validation.py` seems good here?